core/src/main/kotlin/com/google/container/tools/PluginInfo.kt

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2018 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.container.tools
+
+import com.intellij.ide.plugins.IdeaPluginDescriptor
+import com.intellij.ide.plugins.PluginManager
+import com.intellij.openapi.extensions.PluginId
+
+const val CONTAINER_TOOLS_PLUGIN_ID = "com.google.container.tools"
+const val UNKNOWN_VERSION = "N/A"
+
+/** Utilities to get common plugin information such as its version. */
+object PluginInfo {
+
+    /** Returns version of the plugin installed, or `N/A` if version cannot be determined. */
+    fun getPluginVersion(): String {
+        val ideaPluginDescriptor: IdeaPluginDescriptor? =
+            PluginManager.getPlugin(PluginId.getId(CONTAINER_TOOLS_PLUGIN_ID))
+
+        return ideaPluginDescriptor?.version ?: UNKNOWN_VERSION
```

Comment:
I think this is returning the IDE version, the plugin version ?

Reply:
This is what we use in Cloud Tools to get plugin version, for now we don't have a version so probably that's the reason it just takes the IDE version instead.

Reply:
interesting. I'm curious to know how this works. lgtm though if you are saying this will work properly